lte/gateway/docker/services/python/Dockerfile

```diff
@@ -59,11 +59,13 @@ COPY ./lte/gateway/configs $MAGMA_ROOT/lte/gateway/configs
 COPY ./orc8r/gateway/configs/templates $MAGMA_ROOT/orc8r/gateway/configs/templates
 
 COPY ./protos $MAGMA_ROOT/protos
+COPY ./cwf/protos $MAGMA_ROOT/cwf/protos
 COPY ./lte/protos $MAGMA_ROOT/lte/protos
 COPY ./orc8r/protos $MAGMA_ROOT/orc8r/protos
 COPY ./feg/protos $MAGMA_ROOT/feg/protos
 COPY ./dp/protos $MAGMA_ROOT/dp/protos
 
+COPY cwf/swagger $MAGMA_ROOT/cwf/swagger
```

Comment:
Should this not be `COPY ./cwf/swagger ...`?

Reply:
This probably doesn't make a difference anyway, but let's keep it consistent.

Reply:
Yes it does not change anything - but I pushed the change.